### PR TITLE
Feature bot 749 message arrays required

### PR DESCRIPTION
**Pull request by kun2985**
# Description

This PR add validator to override the default validator message "[] is too short"
## Issue (optional)

Issue link: bot-749

## How to test

Here are the following steps to test this pull request:

- POST / PUT disruption and POST / PUT impact with "impacts":[]
- POST / PUT disruption and POST / PUT impact with "objects":[]
- POST / PUT disruption and POST / PUT impact with "application_periods":[]

### CODE REVIEW

File: chaos/validate_params.py [hunk 1]
```diff
@@ -177,3 +177,33 @@ def wrapper(*args, **kwargs):
                     ), 400
             return func(*args, **kwargs)
         return wrapper
+
+# To override the default validator message "[] is too short"
+class validate_required_arrays(object):
```
Comment (pthegner) [inline chaos/validate_params.py]: If it only concern array of type impact, you should rename it :rofl:

Comment (pthegner) [reply chaos/validate_params.py]: like validate_required_arrays_of_impact

Comment (dvdn) [reply chaos/validate_params.py]: _impact's'_ or _application_periods_ or _objects_ type arrays

**Comment by pthegner** [comment]
see PR https://github.com/CanalTP/Chaos/pull/392 for an alternative (tests to add)

**Comment by pthegner** [comment]
Refacto on https://github.com/CanalTP/Chaos/pull/392

**Pull request closed without merging**